pandas/tests/plotting/test_frame.py

```diff
@@ -231,14 +231,40 @@ def test_plot_xy(self):
     @pytest.mark.slow
     def test_logscales(self):
         df = DataFrame({'a': np.arange(100)}, index=np.arange(100))
+
         ax = df.plot(logy=True)
         self._check_ax_scales(ax, yaxis='log')
+        assert ax.get_yscale() == 'log'
+
+        ax = df.plot(logy='sym')
+        self._check_ax_scales(ax, yaxis='symlog')
+        assert ax.get_yscale() == 'symlog'
 
         ax = df.plot(logx=True)
         self._check_ax_scales(ax, xaxis='log')
+        assert ax.get_xscale() == 'log'
+
+        ax = df.plot(logx='sym')
+        self._check_ax_scales(ax, xaxis='symlog')
+        assert ax.get_xscale() == 'symlog'
 
         ax = df.plot(loglog=True)
         self._check_ax_scales(ax, xaxis='log', yaxis='log')
+        assert ax.get_xscale() == 'log'
+        assert ax.get_yscale() == 'log'
+
+        ax = df.plot(loglog='sym')
+        self._check_ax_scales(ax, xaxis='symlog', yaxis='symlog')
+        assert ax.get_xscale() == 'symlog'
+        assert ax.get_yscale() == 'symlog'
+
+    @pytest.mark.parametrize("wrong_input", ["sm", "symlog"])
+    def test_invalid_logscale(self, wrong_input):
+        df = DataFrame({'a': np.arange(100)}, index=np.arange(100))
+
+        msg = "Valid inputs are boolean, None and 'sym'."
+        with pytest.raises(ValueError, match=msg):
+            df.plot(logy=wrong_input)
```

Comment:
might be nice to parametrize over the other possible inputs that can be invalid too, something along the lines of:

```python
@pytest.mark.parametrize("wrong_input", ["sm", "symlog"])
@pytest.mark.parametrize('input_param', ['logx', 'logy', 'loglog'])
def test_invalid_logscale(self, wrong_input, input_param):
    ...
    df.plot(**{input_param: wrong_input})
```

Reply:
cool!! thanks!